admin/dump_pdgid_to_corsika7.py

```diff
@@ -92,18 +92,18 @@
     (157, common_particles["Omega_c_0_bar"]),
     (119, common_particles["D_0_bar"]),
     (120, common_particles["D_s_plus"]),
-    (161, common_particles["Sigma_c_2455_pp"]),
+    (161, common_particles["Sigma_c_2520_pp"]),
```

Comment:
Are you sure of this and similar changes? As I know nothing about Corsika, where can these mappings be checked?

I'm doubting because I then see no matching for the lowest level c-baryons and only for excited ones. So (see https://pdglive.lbl.gov/ParticleGroup.action?init=0&node=BXXX040) the Sigma_c ground level is identified by the mass 2455, the excited Sigma_c^* one with 2520, and the Xi_c^* with 2645.

I let you comment and check it all as you have all info in hands :). Let me know. Thanks a lot.

Reply:
You can check the Corsika7IDs in the wonderful and very comprehensive Corsika userguide on page 120 and 121 (pdf page 132 and 133) https://web.iap.kit.edu/corsika/usersguide/usersguide.pdf

The Sigma_c (2455) states are in the script with IDs 140-142 and 152-154. (Eventually I should print the list in an ordered version... 😅)

Thanks for going throught the PR and helping me improve (Particle as well as my own coding style)! :)